ProcessLib/Parameter/FunctionParameter.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2019, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "FunctionParameter.h"
+
+#include "BaseLib/ConfigTree.h"
+#include "MeshLib/Mesh.h"
+
+namespace ProcessLib
+{
+std::unique_ptr<ParameterBase> createFunctionParameter(
+    std::string const& name, BaseLib::ConfigTree const& config,
+    MeshLib::Mesh const& mesh)
+{
+    //! \ogs_file_param{prj__parameters__parameter__type}
+    config.checkConfigParameter("type", "Function");
+
+    std::vector<std::string> vec_expressions;
+
+    //! \ogs_file_param{prj__parameters__parameter__Function__expression}
```

Comment:
the tag documentation files are missing. See https://github.com/ufz/ogs/tree/master/Documentation/ProjectFile/prj/parameters/parameter

Reply:
done